### ISSUE TITLE
rev748, TPF Tools, Size Ratio & MIPs

### ISSUE THREAD

**Issue by starkgate** [ME3Explorer/ME3Explorer#298]
Autofix stalls when creating MIPs for textures that have a size ratio different from 1x1 or 2x1.

Testing files : http://filebin.ca/2cxhOo2lyWxT

**Comment by starkgate** [comment]
Another issue I noticed, that may or may not be related to this one :
Some textures (only 2x1 V8U8 from my testing, but it may apply to others), can't be autofixed for MIPs. If I autofix, extract then load again, here is what I get :
![capture](https://cloud.githubusercontent.com/assets/12874678/14365327/627b11ce-fd0d-11e5-92c7-19c4eb2318f9.PNG)
rev718 _does_ detect the autofixed MIPs though, and so does the DDS Checker :

Output for rev748 autofix :

```
magic               : 0x20534444 - DDS 
dwSize          : 0x0000007C - 124
dwFlags         : 0x00021007 - [DDSD_CAPS] [DDSD_HEIGHT] [DDSD_WIDTH] [DDSD_PIXELFORMAT] [DDSD_MIPMAPCOUNT] 
dwHeight            : 0x00000200 - 512
dwWidth         : 0x00000400 - 1024
dwPitchOrLinearSize : 0x00000000 - 0
dwDepth         : 0x00000000 - 0
dwMipMapCount       : 0x0000000A - 10
dwReserved1
                    00 = 0x00000000 - 00000000  
                    01 = 0x00000000 - 00000000  
                    02 = 0x00000000 - 00000000  
                    03 = 0x00000000 - 00000000  
                    04 = 0x00000000 - 00000000  
                    05 = 0x00000000 - 00000000  
                    06 = 0x00000000 - 00000000  
                    07 = 0x00000000 - 00000000  
                    08 = 0x00000000 - 00000000  
                    09 = 0x00000000 - 00000000  
                    10 = 0x00000000 - 00000000  
ddspf.dwSize        : 0x00000020 - 32
ddspf.dwFlags       : 0x00080000 - 
ddspf.dwFourCC      : 0x00000000 - 
```

Output for Photoshop DDS :

```
magic               : 0x20534444 - DDS 
dwSize          : 0x0000007C - 124
dwFlags         : 0x000A1007 - [DDSD_CAPS] [DDSD_HEIGHT] [DDSD_WIDTH] [DDSD_PIXELFORMAT] [DDSD_MIPMAPCOUNT] [DDSD_LINEARSIZE] 
dwHeight            : 0x00000200 - 512
dwWidth         : 0x00000400 - 1024
dwPitchOrLinearSize : 0x00100000 - 1048576
dwDepth         : 0x00000000 - 0
dwMipMapCount       : 0x0000000B - 11
dwReserved1
                    00 = 0x00000000 - 00000000  
                    01 = 0x00000000 - 00000000  
                    02 = 0x00000000 - 00000000  
                    03 = 0x00000000 - 00000000  
                    04 = 0x00000000 - 00000000  
                    05 = 0x00000000 - 00000000  
                    06 = 0x00000000 - 00000000  
                    07 = 0x00000000 - 00000000  
                    08 = 0x00000000 - 00000000  
                    09 = 0x00000000 - 00000000  
                    10 = 0x00000000 - 00000000  
ddspf.dwSize        : 0x00000020 - 32
ddspf.dwFlags       : 0x00080000 - 
ddspf.dwFourCC      : 0x00000000 - 
```

Test files : http://filebin.ca/2cyKaUFs0xkp

**Comment by ghost** [comment]
Verified in rev749 prebuild.

**Comment by ghost** [comment]
For future reference, it's possible that this fix broke #212.
